pkg/sql/sqlerrors/errors.go

```diff
@@ -52,6 +53,24 @@ func NewSchemaChangeOnLockedTableErr(tableName string) error {
 			"\"ALTER TABLE %v SET (schema_locked = true);\"", tableName, tableName)
 }
 
+// NewDisallowedSchemaChangeOnLDRTableErr creates an error that indicates that
+// the schema change is disallowed because the table is being used by a
+// logical data replication job.
+func NewDisallowedSchemaChangeOnLDRTableErr(tableName string, jobIDs []int64) error {
+	ids := make([]string, len(jobIDs))
+	for i, v := range jobIDs {
+		ids[i] = strconv.Itoa(int(v))
+	}
+	return errors.WithHintf(
+		pgerror.Newf(
+			pgcode.FeatureNotSupported,
+			"this schema change is disallowed on table %s because it is referenced by "+
+				"one or more logical replication jobs [%s]", tableName, strings.Join(ids, ", "),
+		),
+		"To perform the schema change, cancel the logical replication job, then execute "+
```

Comment:
hrm, for certain schema changes, this is the right advice, but for others, the sc may make LDR impossible resume. How about we remove this directive for now, and rely on docs for how a user should run an unsupported SC?

Reply:
sounds good!